File: project-program2/core/src/com/mygdx/game/MyGdxGame.java
package com.mygdx.game;

import com.badlogic.gdx.Game;
import com.badlogic.gdx.Gdx;
import com.badlogic.gdx.assets.AssetManager;
import com.badlogic.gdx.graphics.GL20;
import com.badlogic.gdx.graphics.Texture;
import com.badlogic.gdx.graphics.g2d.SpriteBatch;
import com.mygdx.game.screens.MainMenu;
import com.mygdx.game.screens.SimScreen;

public class MyGdxGame extends Game {
	public static Integer SCREENS = 1;

	public SpriteBatch batch;
	private AssetManager assetManager;
	Texture img;


	@Override
	public void create () {
		batch = new SpriteBatch();
		//img = new Texture("badlogic.jpg");
		setScreen(new MainMenu(this, this));
		System.out.println("windows open");
	}

	@Override
	public void render () {
		super.render();
	}

	@Override
	public void dispose () {
		batch.dispose();

	}
}

File: project-program2 - kotlin/core/src/com/mygdx/game/MyGdxGame.kt
package com.mygdx.game

import com.badlogic.gdx.Game
import com.badlogic.gdx.assets.AssetManager
import com.badlogic.gdx.graphics.Texture
import com.badlogic.gdx.graphics.g2d.SpriteBatch
import com.mygdx.game.screens.MainMenu

class MyGdxGame : Game() {
    @JvmField
	var batch: SpriteBatch? = null
    private val assetManager: AssetManager? = null
    var img: Texture? = null
    override fun create() {
        batch = SpriteBatch()
        //img = new Texture("badlogic.jpg");
        setScreen(MainMenu(this, this))
        println("windows open")
    }

    override fun render() {
        super.render()
    }

    override fun dispose() {
        batch!!.dispose()
    }

    companion object {
        var SCREENS = 1
    }
}